source/04-measurement.Rmd

```diff
@@ -321,7 +321,7 @@ The outcomes can be labelled by *any symbols of your choice* --- it is the *deco
 This said, labelling outcomes with real numbers is very useful.
 Some textbooks describe observables in terms of Hermitian operators, claiming that the corresponding operators *have to* be Hermitian "because the outcomes are real numbers". This is actually a bit backwards. As we say above, the labels can be arbitrary, but, since real number labels are often useful (as we're about to justify), we tend to only work with Hermitian operators.
 
-For example, the **expected value** $\av{A}$ (also known as the **mean**), which is the average of the numerical values $\lambda_k$ weighted by their probabilities, is a very useful quantity and can be easily expressed in terms of the operator $A$ and the state of the system $\ket{\psi}$ as follows:
+For example, the **expected value** $\av{A}$ (also known as the **mean**), which is the average of the numerical values $\lambda_k$ weighted by their probabilities, is a very useful quantity and can be easily expressed in terms of the operator $A$ and the state of the system $\ket{\psi}$ as follows:^[It is important to note here that $\av{A}$ is dependent upon the initial state in which we prepare our quantum system, and is sometimes denoted $\av{A}_{\psi}$. This dependency on the initial state is often tacitly implied in the notation of the literature.]
```

Comment:
Good thing to point out! Could you please change the footnote to the following: "It is important to note here that the notation $\av{A}$ is slightly misleading, since it omits the dependence on the initial state $\ket{\psi}$. Some authors thus write $\av{A}_{\ket{\psi}}$ instead, but many opt (as we do) for the more succinct notation."

Reply:
(just trying to keep the tone of the book consistent with how I would probably write this!)

Reply:
Definitely, will do! Thanks for the edits :)